views-thymeleaf/src/main/java/io/micronaut/views/thymeleaf/LinkBuilder.java

```diff
@@ -32,6 +33,12 @@
 @Singleton
 public class LinkBuilder extends StandardLinkBuilder {
 
+    private final String contextPath;
+
+    public LinkBuilder(@Nullable @Property(name = "micronaut.server.context-path") String contextPath) {
```

Comment:
This is a breaking change, could you restore the original constructor and annotate this one with `@Inject`

A test would be nice too